packages/react-components/src/components/DatePicker/DatePicker.mdx

```diff
@@ -0,0 +1,27 @@
+import { Canvas, Story, ArgsTable, Meta, Title } from '@storybook/blocks';
+import { Figma } from "@storybook/addon-designs/blocks";
+import * as DatePickerStories from './DatePicker.stories';
+import { BlockColumnsContainer, BlockColumns } from '../../stories/components/BlockColumns';
+
+<Meta of={DatePickerStories} />
+
+<Title>DatePicker</Title>
+
+[Component API](#ComponentAPI) | [Content Spec](#ContentSpec)
+
+<Canvas of={DatePickerStories.DatePicker} sourceState="none"/>
+
+#### Example implementation
+```jsx
+<DatePicker
+  fromMonth={new Date('06/20/2021')}
+  toMonth={new Date('09/20/2021')}
+  month={new Date('09/20/2021')}
+/>
+```
+
+## Component API <a id="ComponentAPI" />
+<ArgsTable />
```

Comment:
The Component API here shows only two params: `innerRef` and `range`. I think you can use here a `<ArgTypes of={DatePickerStories.DatePicker} />` to show props related to this basic story